style.css

```diff
@@ -518,3 +538,222 @@ textarea {
   place-items: center;
   margin-top: 20px;
 }
+
+@media screen and (min-width: 768px) {
+  .cont-list {
+    display: flex;
+    gap: 55px;
+    margin-right: 45px;
+    list-style: none;
+    margin-left: 60%;
+  }
+
+  .cont-list li a {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .nav-logo {
+    margin-left: 10%;
+  }
+
+  /* .introduction p{
+  width: 50%;
+} */
+
+  .hambuger {
+    display: none;
+  }
+
+  .pic {
+    background-image: url("images/bg.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding-top: 249px;
+    padding-left: 260px;
+    padding-right: 260px;
+  }
+
+  .intro-card {
+    padding-left: 1%;
+  }
+
+  .work-card {
+    flex-direction: row;
+    width: 750px;
+    gap: 30px;
+    justify-content: space-between;
+  }
```

Comment:
- Hey, the width of each card is small as compared to the Figma design. You should use a fluid unit of measurement for the width like % so that it would be responsive to the device width. As of now, your card are fixated due to the fixed value in pixels you've used. please, adjust the size of your cards. see the image below:
- let's improve the spacing between the button to the end of the card and the space on the Tonic to match the design as much as you can. 
- The image on the desktop version is big while yours is small as compared to the Figma design. please, let's fix this issue to improve our design.

![Screenshot from 2023-02-23 22-07-57](https://user-images.githubusercontent.com/59397159/221031650-2c08af3a-9b09-42e3-a22a-1ef4304fb846.png)
![Screenshot from 2023-02-23 22-07-38](https://user-images.githubusercontent.com/59397159/221031679-92c30bda-c819-45c9-a232-3e4139def706.png)